Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/view/components/SoundComponent.java

```diff
@@ -0,0 +1,109 @@
+package pg.autyzm.friendly_plans.view.components;
+
+import android.content.Context;
+import android.media.AudioManager;
+import android.media.MediaPlayer;
+import android.view.View;
+import android.view.animation.Animation;
+import android.view.animation.AnimationUtils;
+import android.widget.ImageView;
+import database.entities.Asset;
+import database.repository.AssetRepository;
+import java.io.IOException;
+import javax.inject.Inject;
+import pg.autyzm.friendly_plans.AppComponent;
+import pg.autyzm.friendly_plans.R;
+import pg.autyzm.friendly_plans.asset.AssetsHelper;
+import pg.autyzm.friendly_plans.notifications.ToastUserNotifier;
+
+public final class SoundComponent {
```

Comment:
I really like this, extracting is always rewarding! :1st_place_medal:

Reply:
If we have so beautiful class now, it would be worth to add some tests to it 🌵

Reply:
Great idea - I'll try to do something with it :)